dts/arm64/qemu/qemu-virt-a55.dtsi

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+ * Copyright 2022 NXP
```

Comment:
I believe on all these files only NXP copyright should remain, even if you used those from baylibre as examples.

Reply:
Some of them were not just used as examples but simply copied (e.g: mmu_regions.c). If you still believe I should remove their original copyrights with the NXP one I will update the PR.

Reply:
Well...for cases like copy-paste we should find a solution to use the same file, not duplicate the code - maybe you can do this for mmu_regions.c. 
For the others, I believe you can just use NXP.

Reply:
I agree it's redundant but if we were to use the same code the following problems could appear:

1) Which files should we reference from our board? (e.g: in the case of mmu_regions.c, should we use the one from qemu_cortex_a53 or qemu_virt_arm64 (they are the same file))
2) What happens if the referenced file gets modified?

If we were to add mmu_regions.c for example to a common folder I don't think that would be worth the effort since only a couple of boards use the same mmu_regions.c. Also if we were to take this approach for all the boards that use common files this might not scale very well. Also, if we were to do this, we should definitely submit another PR as I don't think it makes the object of the current one.

Reply:
ok, leave it as is, now.
Let's put this upstream and we'll see later